RELEASES.md

```diff
@@ -233,7 +233,7 @@ instructions in the AMD ROCm documentation.
 
 <!-- TODO: clean up these sections and confirm the instructions work -->
 
-Here's a quick way assuming you copied the all the tar files into `${BUILD_ARTIFACTS_DIR}` to "install" TheRock into `${BUILD_ARTIFACTS_DIR}/output_dir`
+Here's a quick way assuming you copied all the tar files into `${BUILD_ARTIFACTS_DIR}` to "install" TheRock into `${BUILD_ARTIFACTS_DIR}/output_dir`
```

Comment:
I'm completely reworking this section in my next PR (https://github.com/ROCm/TheRock/pull/813), fyi.